master/reference/calicoctl/resources/networkpolicy.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Network Policy Resource (NetworkPolicy)
+---
+
+A Network Policy resource (NetworkPolicy) represents an ordered set of rules which are applied
```

Comment:
longer term, @emanic may be able to refactor GlobalNetworkPolicy and NetworkPolicy since they share high level descriptions